benchmarks/src/test/java/org/apache/druid/benchmark/BenchmarkDataGeneratorTest.java

```diff
@@ -431,19 +431,19 @@ public void addRow(InputRow row)
     public void printStuff()
     {
       System.out.println();
-      for (String dim : dimensionMap.keySet()) {
+      for (Map<Object, Integer> dim : dimensionMap.values()) {
         System.out.println("DIMENSION " + dim + "\n============");
-        Map<Object, Integer> valueMap = dimensionMap.get(dim);
+        Map<Object, Integer> valueMap = dim;
 
         List<Comparable> valList = new ArrayList<>();
-        for (Object val : valueMap.keySet()) {
+        for (Integer val : valueMap.values()) {
           valList.add((Comparable) val);
         }
 
         Collections.sort(valList);
 
         for (Comparable val : valList) {
-          System.out.println(" VAL: " + val + " CNT: " + valueMap.get(val));
+          System.out.println(" VAL: " + val + " CNT: " + val);
```

Comment:
Logic is changed. Now prints `val` and `val`. Maybe easier to rewrite this code by using `new TreeMap(valueMap)` and then iterating that map, instead of moving to an ArrayList and sorting.

Reply:
So what this function is actually doing? Will this suffice ?
`  
    for(Integer val: valueMap.values() {
          System.out.println(" VAL: " + val + " CNT: " + val);
        }
`

Reply:
It sorts entries first. You can do
```java
new TreeMap<>(valueMap).forEach((val, cnt) -> System.out.println(" VAL: " + val + " CNT: " + cnt));